Mohammad Idham

midham14 opened the pull request:
1. Add Register Server
2. Add Login Server
3. Add Get Tasks Server
4. Add Post Tasks Server
5. Add Edit Tasks Server
6. Add Delete Tasks Server
7. add api-documentation
8. add client login,register,logout,googlelogin 
9. add client crud add task, edit category, delete task, get task
10. add server google login


client deploy : https://kanban-idham.web.app/
server deploy : https://kanban-idham.herokuapp.com/

midham14 committed:
1. Add Register Server
2. Add Login Server
3. Add Get Tasks Server
4. Add Post Tasks Server
5. Add Edit Tasks Server
6. Add Delete Tasks Server
7. add api-documentation

ziadyjo commented on server/app.js:
```diff
@@ -0,0 +1,28 @@
+require('dotenv').config()
```
nanti kalau deploy kasih conditional ya

ziadyjo commented on server/controllers/taskController.js:
```diff
@@ -0,0 +1,62 @@
+
+
+class TaskController{
+    static getTaskHandler(req,res,next){
```
get tasknya aman ya? nanti difilter di client

ziadyjo commented on server/middlewares/authentication.js:
```diff
@@ -0,0 +1,32 @@
+const { User } = require('../models')
```
authentication aman yaa

ziadyjo commented on server/middlewares/authorization.js:
```diff
@@ -0,0 +1,24 @@
+const { Task } = require('../models')
```
authorization aman yaa

ziadyjo reviewed (commented):
Udah lumayan yaa 👍🏻

midham14 committed:
1. add client login,register,logout,googlelogin
2. add client crud add task, edit category, delete task, get task
3. add server google login
note : belum setting deploy

midham14 committed:
server deploy setting

midham14 committed:
client deploy setting

midham14 committed:
deployed
